scxml_converter/src/scxml_converter/scxml_entries/scxml_ros_action_client.py

```diff
@@ -124,6 +124,15 @@ def check_validity(self) -> bool:
                                            self._reject_target)
         return valid_name and valid_accept and valid_reject
 
+    def instantiate_bt_events(self, _: str):
+        # We do not expect a body with BT ports to be substituted
+        pass
+
+    def update_bt_ports_values(self, _) -> None:
+        """Update the values of potential entries making use of BT ports."""
+        # We do not expect a body with BT ports to be substituted
+        pass
+
```

Comment:
I think we also need this for the other declarations, right?

Reply:
Not in here, this expects no executable body in the related scxml tag, hence no BT port substitution